types/superagent/superagent-tests.ts

```diff
@@ -282,8 +282,7 @@ request
     .get("/blob")
     .responseType("blob")
     .end((err, res) => {
-        assert(res.xhr instanceof XMLHttpRequest);
-        assert(res.xhr.response instanceof Blob);
+        console.log(res.xhr.response);
```

Comment:
```suggestion
        res.xhr; // $ExpectType any
        res.xhr.response; // $ExpectType any
```
it's equivalent of runtime assertions, but checks (asserts) current types using tsc only